chapter2/tests/t14/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10.11-slim-buster AS builder
+
+WORKDIR /app
+COPY memory.py requirements.txt ./
+RUN apt-get update && apt-get install -y gcc=4:8.3.0-1 python3-dev=3.7.3-1 musl-dev=1.1.21-2 --no-install-recommends
+RUN pip install --no-cache-dir -r requirements.txt --upgrade pip && pip install --no-cache-dir -r requirements.txt
+
+COPY . .
+
+FROM gcr.io/distroless/python3@sha256:c472650ee9b43ce1bb7efa2960c95c046fa44f7258ae22f9ceb3d3f49f26d010 AS runtime
```

Comment:
use human tags, not sha commits.

Reply:
> 

@dmitry-mightydevops 
I did not figure out how to find a specific tag for this image correctly, there are no tags on the Google website except for "latest", but "latest" is not c
![Screenshot from 2023-04-28 01-16-26](https://user-images.githubusercontent.com/93929010/235003289-574398ae-d1ff-4462-9c39-1a10e0fd23bc.png)
orrect to use

Reply:
@rtest12 
```
curl https://gcr.io/distroless/python3/tags
```
![image](https://user-images.githubusercontent.com/70822710/235247954-bc63c796-4b1e-4c4b-b541-8bfca269a5f6.png)